lib/net/queries/actions/useAddComment.ts

```
@@ -53,6 +52,10 @@ const useAddComment = () => {
         bodyParams
       );
       return response.data;
+      return {
+        message: "Comment added successfully",
+        commentId: "12345",
+      };
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)